subprojects/docs/docs.gradle

```
@@ -550,6 +519,16 @@ samples {
             }
         }
 
+        publishingConventionPlugins {
+            sampleDirectory = samplesRoot.dir("build-organization/publishing-convention-plugins")
+            displayName = "Sharing configuration between projects in multi-repo setup"
```

```suggestion
            displayName = "Sharing build logic in a multi-repo setup"
```

This currently renders as two lines. Hence the suggestion to shorten it.